test/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
 	<PropertyGroup>
+		<TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
line 17 will never be evaluated with this change

Reply:
Thanks for the call out, I've removed this line